src/Authentication/Controllers/SystemUserController.cs

```diff
@@ -16,7 +16,7 @@ namespace Altinn.Platform.Authentication.Controllers
     /// <summary>
     /// CRUD API for the System User 
     /// </summary>
-    [Authorize]
+    //[Authorize]
```

Comment:
commented on purpose_

Reply:
probably a merge relic, will remove the comment line